### ISSUE TITLE
make links dragable and able to attach events

### ISSUE THREAD

**Issue by teseo** [bevacqua/dragula#150]
Hello,

First of all I would like to congratulate you for such a fantastic plugin. Very helpful.

I was wondering wether is possible or is a nice feature to have, attaching events to a 'A' TagName.

We aim to be able to drag elements between containers through a 'A' tagName and we could accomplish it by using the invalid option or use moves and setup 'A' tagName as handler.

Do you think is possible to be able to attach click, mousedown, mouseup, etc. events so we can control wether the user was willing to drag and drop or wants to have the default navigation behaviour to the A's href.

¡Muchas gracias!

**Comment by bevacqua** [comment]
Refer to the docs: https://github.com/bevacqua/dragula#optionsinvalid

**Comment by teseo** [comment]
Hello,

Many thanks for your quick response. I might not have made myself clear on my previous post. Sorry about that.

The option Invalid allows me to be able to drag from 'A' tagNames. We have made that and is working great. 

The issue is that when you click in a 'A' tagName, with the invalid options below, I am not able to attach any events to the link whether is by class name of by Id. 

I was wondering if it is possible to have this:
```
function invalidTarget (el) {
  return el.tagName === 'BUTTON';
}
```
and being able to attach events to container elemtents with a 'A' tagName.


Again many thanks for your plugin.

**Comment by bevacqua** [comment]
Not sure what you mean by attach events.

**Comment by teseo** [comment]
Hello, 

Again many thanks for your response. By attaching events I mean the capacity when a user perform an action and generates an event (like click, keydown, mouseover, keyup, etc.) on the link, being able to perform an action. 

In the dragula case, when you make an A tagname dragable, you lose the default behaviour of the link (go whereever the href is taking you) and also I can't attach the click event or any other as dragula seems to hijack any event on dragable handlers.

**Comment by bevacqua** [comment]
This will be fixed in 3.0.0

**Comment by teseo** [comment]
Wonderful!

We can consider this thread as closed so whoever is in the same situation I am, can see that it will be covered in 3.0.0

¡thanks!
